gmail/modules/listener/Dispatcher.bal

```diff
@@ -148,13 +143,13 @@ class Dispatcher {
     }
 
     isolated function dispatchNewLabeled(gmail:HistoryEvent addedlabel) returns @tainted error? {
```

Comment:
Shall we update method name as `dispatchNewLabel`?

Reply:
+1